aviary/variable_info/variables.py

```diff
@@ -604,61 +601,72 @@ class Wing:
 
 
 class Dynamic:
-    """Dynamic mission data hierarchy"""
+    """All time-dependent variables used during mission analysis"""
+
+    class Atmosphere:
+        """Variables related to atmospheric/freestream conditions"""
```

Comment:
I would just say this is "atmospheric & freestream" conditions since Mach, Mach_rate, Velocity, and Velocity_rate, are describing the free stream conditions. My initial impulse was to put these back in Dynamic/Mission

Reply:
It's hard for me to think of `Dynamic.Atmosphere.ALTITUDE` as different from a vehicle state, since it is literally the altitude state of the vehicle.